crates/types/src/sensor_data.rs

```diff
@@ -7,9 +8,9 @@ use crate::joints::Joints;
 #[derive(Clone, Copy, Debug, Default, Deserialize, Serialize, SerializeHierarchy)]
 pub struct InertialMeasurementUnitData {
     // Linear acceleration is coming from a left handed coordinate system
-    pub linear_acceleration: Vector3<f32>,
-    pub angular_velocity: Vector3<f32>,
-    pub roll_pitch: Vector2<f32>,
+    pub linear_acceleration: Vector3<Robot>,
+    pub angular_velocity: Vector3<Robot>,
```

Comment:
I am not sure whether this is correct and/or we want to do that. What do you think @knoellle ?

Reply:
I think if you want to use a vector here, it should be framed accordingly.
IIRC nalgebra just uses tuples for euler angles, but of course that doesn't implement things like addition, uniform s.aling, etc.
I'm fine with this solution.